README.md

```diff
@@ -326,6 +332,22 @@ A list of available keycodes can be found here: [keycodes](https://github.com/mu
     value = "value"
 ```
 
+#### Switch to another deck
```

Comment:
I know this can be a bit painful (and don't worry about it for now), but for the future: I'd appreciate documentation changes and things like the special actions in separate PRs. It keeps the entire review process a lot shorter and requires everyone to context switch less often.

Reply:
OK. I'll try better in the future.  Is it OK to keep the documentation for the particular change in the same PR?

Reply:
Yes, don't worry, I don't want to cause more work on your side for now! Great work!